go.mod

```
@@ -0,0 +1,57 @@
+	github.com/awalterschulze/gographviz v0.0.0-20190221210632-1e9ccb565bca
+	github.com/blang/semver v1.1.1-0.20190414182527-1a9109f8c4a1
+	github.com/certifi/gocertifi v0.0.0-20180905225744-ee1a9a0726d2 // indirect
+	github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96 // indirect
+	github.com/dustin/go-humanize v1.0.0
+	github.com/elazarl/goproxy v0.0.0-20190911111923-ecfe977594f1 // indirect
+	github.com/getsentry/raven-go v0.0.0-20171206001108-32a13797442c // indirect
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
+	github.com/gogo/protobuf v1.2.2-0.20190415061611-67e450fba694 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
+	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367 // indirect
+	github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d // indirect
+	github.com/gorilla/mux v1.7.1
+	github.com/gregjones/httpcache v0.0.0-20170728041850-787624de3eb7 // indirect
+	github.com/hashicorp/golang-lru v0.5.1
+	github.com/imdario/mergo v0.3.5 // indirect
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/json-iterator/go v0.0.0-20180701071628-ab8a2e0c74be // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/onsi/ginkgo v1.10.1 // indirect
+	github.com/onsi/gomega v1.7.0 // indirect
+	github.com/openshift/api v3.9.1-0.20190419161300-aae63d5f0f50+incompatible
+	github.com/openshift/client-go v0.0.0-20190412095722-0255926f5393
+	github.com/openshift/library-go v0.0.0-20190419201117-4bae0f495ea7
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/pkg/profile v1.3.0 // indirect
+	github.com/prometheus/client_golang v0.9.3-0.20190416085848-65d3a96fbaa7
+	github.com/prometheus/common v0.3.1-0.20190416093430-c873fb1f9420 // indirect
+	github.com/prometheus/procfs v0.0.0-20190416084830-8368d24ba045 // indirect
+	github.com/russross/blackfriday v2.0.0+incompatible
+	github.com/satori/go.uuid v1.2.0
+	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
+	github.com/sirupsen/logrus v1.2.0
+	github.com/spf13/cobra v0.0.2-0.20180319062004-c439c4fa0937
+	github.com/spf13/pflag v1.0.1 // indirect
+	go.opencensus.io v0.22.1 // indirect
+	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
+	golang.org/x/time v0.0.0-20181108054448-85acf8d2951c
+	google.golang.org/api v0.10.0
+	gopkg.in/inf.v0 v0.9.0 // indirect
+	k8s.io/api v0.0.0-20190222213804-5cb15d344471
+	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
+	k8s.io/client-go v0.0.0-20190228174230-b40b2a5939e4
+	k8s.io/klog v0.0.0-20181108234604-8139d8cb77af
+	k8s.io/kube-openapi v0.0.0-20181109181836-c59034cc13d5 // indirect
+	sigs.k8s.io/yaml v1.1.0 // indirect
+)
```

@stevekuznetsov If you add the following block, it will resolve the **log_dir** issue that is occurring with your changes:
```suggestion
)

replace (
          github.com/golang/glog => github.com/openshift/golang-glog 3c92600d7533018d216b534fe894ad60a1e6d5bf
)
```
Run a `tidy` and a `vendor` to pick up all the latest/greatest changes, and the release-controller should be operational again.